src/summary/history/encounter.resource.tsx

```diff
@@ -0,0 +1,13 @@
+import { openmrsFetch } from "@openmrs/esm-api";
+import { mockPatientEncounterSearchResponse } from "../../../__mocks__/encounters.mock";
+
+export function encounterResource(
+  patientIdentifer: string,
+  abortController: AbortController
+) {
+  return openmrsFetch(
+    `/ws/fhir/Encounter?patient.identifier=${patientIdentifer}`,
+    { signal: abortController.signal }
+  );
+  // return Promise.resolve(mockPatientEncounterSearchResponse);

```

Comment:
Will merging this in break things? Does the fhir endpoint for encounters currently work?

We should always merge code in that works. If the fhir endpoint isn't ready, we should use the mocked data for now. We should not be merging in commented code.